nginx/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,136 @@
+user nginx;
+worker_processes 4;
+pid /run/nginx.pid;
```

Comment:
When I opened up the container image, the default pointed to `/var/run/nginx.pid`. Technically they're the same because I think `/run` is just a symlink to `/var/run`, or vice-versa.